packages/desktop/src-electron/app.ts

```diff
@@ -401,12 +401,12 @@ function createMainWindow() {
     (details, callback) => {
       const { url } = details;
       if (
-        url.startsWith('http://127.0.0.1:21320') ||
-        url.startsWith('http://localhost:21320')
+        url.startsWith('http://127.0.0.1:21320/') ||
+        url.startsWith('http://localhost:21320/')
       ) {
         // resolve onekey bridge CORS error
         details.requestHeaders.Origin = 'https://jssdk.onekey.so';
-      } else if (url.startsWith('https://mainnet.optimism.io')) {
+      } else if (url.startsWith('https://mainnet.optimism.io/')) {
```

Comment:
![WX20230809-202910](https://github.com/OneKeyHQ/app-monorepo/assets/5107241/527f13e8-a229-4af2-99be-0719306c9877)

I can confirm this doesn't affect the result. Electron always gives full URL (with trailing slash if no extra path in the URL).